<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleInstructionTestPage2</class>
 <widget class="QWidget" name="VehicleInstructionTestPage2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="VITest2_Button_prePage">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
background-color: rgb(179, 179, 179);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_nextPage">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(179, 179, 179);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L2R1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU1复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L3R4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>260</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VITest2_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>440</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(179, 179, 179);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L2R2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU2复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L1R1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU1切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L1R3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU3切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L2R3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU3复位</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L3R2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L1R4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU4切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L1R2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU2切除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_closePage">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>370</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 18px ;
background-color: rgb(179, 179, 179);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L3R1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L2R4">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>150</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>ACU4复位</string>
   </property>
  </widget>
  <widget class="QLabel" name="VITest2_Label_Page">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="VITest2_Button_L3R3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75  18px ;
background-color: rgb(255, 192, 0);
border:3px solid rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16px ;
</string>
   </property>
   <property name="text">
    <string>复位按键抬起前
请不要离开该页面</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>VITest2_background</zorder>
  <zorder>VITest2_Button_prePage</zorder>
  <zorder>VITest2_Button_nextPage</zorder>
  <zorder>VITest2_Button_L2R1</zorder>
  <zorder>VITest2_Button_L3R4</zorder>
  <zorder>VITest2_Button_L2R2</zorder>
  <zorder>VITest2_Button_L1R1</zorder>
  <zorder>VITest2_Button_L1R3</zorder>
  <zorder>VITest2_Button_L2R3</zorder>
  <zorder>VITest2_Button_L3R2</zorder>
  <zorder>VITest2_Button_L1R4</zorder>
  <zorder>VITest2_Button_L1R2</zorder>
  <zorder>VITest2_Button_closePage</zorder>
  <zorder>VITest2_Button_L3R1</zorder>
  <zorder>VITest2_Button_L2R4</zorder>
  <zorder>VITest2_Label_Page</zorder>
  <zorder>VITest2_Button_L3R3</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
